routes/message.routes.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express");
+const router = express.Router();
+const { isAuthenticated } = require("../middleware/auth.middleware");
+
+const Message = require("../models/Message.model");
+
+
+
+
+
+
+/**
+api/messages?game=gameRoomId&last=timestamp	GET	display list of messages for room	
+ */
+
```

Comment:
if timestamp is provided, only return messages that were written after the timestamp